monkey/agent_plugins/payloads/cryptominer/Pipfile

```diff
@@ -0,0 +1,11 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
```

Comment:
This will need psutil, though we don't have to add it here until the sha256 task.

Either add it now or add a subtask so we don't forget.